content/en/Platform Deep Dive/Pentests/Reports/_index.md

```diff
@@ -10,17 +10,19 @@ description: >
 Download a pentest report to view security issues that our pentesters found.
 {{% /pageinfo %}}
 
-<p>{{% min-credits-report %}}.</p>
-
-We do not create multiple pentest reports for large [assets](/getting-started/assets/). For example, if you want separate pentest reports for different APIs, set up different pentests for each API.
+You can download a report for each pentest that you perform. Report types and their [contents](/platform-deep-dive/pentests/reports/report-contents/) differ for each [pentest type](/platform-deep-dive/pentests/pentest-types/).
 
 ## Pentest Report Types
 
-  | Report Type                   | Description                                                                                                                                                    |
-|-------------------------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------|
-| Customer Letter               | An executive summary of the pentest. May be used as a certificate of completion. Great for external shareholders. Includes:<br>  <br>- Executive Summary <br>- Methodology |
-| Attestation Report            | Adds the following information to the customer letter: <br> <br>- Pentester user information <br>- An overall list of findings                                                                     |
-| Attestation Letter            | Includes the [executive summary](./report-contents/#executive-summary) as a formal letter, suitable for external stakeholders or customers                                                                     |
-| Full Report                   | Includes the following report sections, beyond attestation:<br>  <br>- Executive Summary <br>- Scope of Work <br>- Methodology <br>- Summary of Findings <br>- Recommendations <br>- Post-Test Remediation     |
-| Full Report + Finding Details | Adds details of every test finding to the full report. Details include:<br>  <br>- Vulnerability Type <br>- Description <br>- Proof of Concept <br>- Severity <br>- Suggested Fix     | 
-| Customized Report             | After you [customize](./customize-report) a report, you'll see this option. You can start from any of the other report templates, except the Attestation Letter. |
+| Report Type | Available for [Pentest Type](/platform-deep-dive/pentests/pentest-types/) | Description | [Customizable](/platform-deep-dive/pentests/reports/customize-report/) |
+|-----|-----|-----|-----|
+| Automated Report | Agile Pentest | A system-generated report for an [Agile Pentest](/getting-started/glossary/#agile-pentest) intended for internal use. Includes the following sections:<br><br><li>Pentester user information</li><li>Executive Summary</li><li>Methodology</li><li>Post-Test Remediation</li><li>Finding Details</li><br>You can download the report once we move the pentest to the _Remediation_ [state](/penteststates/). | Yes |
```

Comment:
@ana-dashuk-cobalt - This ticket captures all sections of the automated report. Even though the once that are identified in the doc are the main ones, please take a look if we need to add more details here.
Additionally, "Pentester user information" => "Pentest information" might read better.